pkg/utils/topology/ticdc.go

```diff
@@ -85,11 +88,14 @@ func parseTiCDCInfo(clusterName string, value []byte) (*TiCDCInfo, error) {
 	}
 
 	return &TiCDCInfo{
-		ClusterName: clusterName,
-		Version:     ds.Version,
-		IP:          hostname,
-		Port:        port,
-		Status:      ComponentStatusUp,
-		StatusPort:  port,
+		ClusterName:    clusterName,
+		GitHash:        ds.GitHash,
+		Version:        ds.Version,
+		IP:             hostname,
+		Port:           port,
+		DeployPath:     ds.DeployPath,
+		Status:         ComponentStatusUp,
+		StatusPort:     port,
```

Comment:
StatusPort is same to Port?

Reply:
Yes, TiCDC uses a single port for external services.